brave-lists/clean-urls.json

```diff
@@ -145,10 +145,14 @@
     },
     {
         "include": [
-            "*://*.twitter.com/*"
+            "*://*.twitter.com/*",
+            "*://*.x.com/*",
```

Comment:
From what I can see in https://chromium.googlesource.com/chromium/src/+/main/extensions/common/url_pattern.cc, `x.com` is already included in `*.x.com` and so we don't need an explicit `x.com`.